zrml/prediction-markets/src/lib.rs

```diff
@@ -2011,26 +2010,6 @@ mod pallet {
     #[pallet::storage_version(STORAGE_VERSION)]
     pub struct Pallet<T>(PhantomData<T>);
 
-    // TODO(#1212): Remove in v0.5.1.
-    #[pallet::storage]
-    pub type MarketIdsPerOpenBlock<T: Config> = StorageMap<
-        _,
-        Blake2_128Concat,
-        T::BlockNumber,
-        BoundedVec<MarketIdOf<T>, CacheSize>,
-        ValueQuery,
-    >;
-
-    // TODO(#1212): Remove in v0.5.1.
-    #[pallet::storage]
-    pub type MarketIdsPerOpenTimeFrame<T: Config> = StorageMap<
-        _,
-        Blake2_128Concat,
-        TimeFrame,
-        BoundedVec<MarketIdOf<T>, CacheSize>,
-        ValueQuery,
-    >;
```

Comment:
This storage item still contains keys on main-net.

Reply:
That's just because we haven't run the update yet.